templates/zerver/help/change-a-user's-name.md

```diff
@@ -0,0 +1,25 @@
+# Change a user's name
+Name of the user can be changed by the admin as well as user himself.
+
+Following are the steps to change the name of the user using an admin account:
```

Comment:
Try to use complete sentences and words in your documentations. How about "Follow the following steps to change the name of a user using an administrator account"?